includes/discovery/cisco-vrf-lite.inc.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+/* Copyright (C) 2014 Nicolas Armando <nicearma@yahoo.com> and Mathieu Millet <htam-net@github.net>
+* This program is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with this program. If not, see <http://www.gnu.org/licenses/>. */
+
+
+global $debug;
+
+// This one only will work with the CISCO-CONTEXT-MAPPING-MIB V2 of cisco
+if ($config['enable_vrf_lite_cisco']) {
+
+    $ids = array();
+
+    // For the moment only will be cisco and the version 3
+    if ($device['os_group'] == "cisco" && $device['snmpver'] == 'v3') {
+
+        echo ("VRF lite cisco : \n");
+        $mib = "SNMP-COMMUNITY-MIB";
+
+        $mib = "CISCO-CONTEXT-MAPPING-MIB";
+        //-Osq because if i put the n the oid from the first command is not the same of this one
+        $listVrf = snmp_walk($device, "cContextMappingVrfName", "-Osq -Ln", $mib, NULL);
+        $listVrf = str_replace("cContextMappingVrfName.", "", $listVrf);
+        $listVrf = str_replace('"', "", $listVrf);
+        $listVrf = trim($listVrf);
+
+        if ($debug) {
+            echo ("\n[DEBUG]\nUsing $mib\n[/DEBUG]\n");
+            echo ("\n[DEBUG List Vrf only name]\n$listVrf\n[/DEBUG]\n");
+        }
+
+        $tableVrf;
+        foreach (explode("\n", $listVrf) as $lineVrf) {
+            $tmpVrf = explode(" ", $lineVrf, 2);
+            //the $tmpVrf[0] will be the context
+            if (count($tmpVrf) == 2 && !empty($tmpVrf[1])) {
+                $tableVrf[$tmpVrf[0]]['vrf_name'] = $tmpVrf[1];
+            }
+        }
+
+        unset($listVrf);
+
+        $listIntance = snmp_walk($device, "cContextMappingProtoInstName", "-Osq -Ln", $mib, NULL);
+        $listIntance = str_replace("cContextMappingProtoInstName.", "", $listIntance);
+        $listIntance = str_replace('"', "", $listIntance);
+        $listIntance = trim($listIntance);
+
+        if ($debug) {
+            echo ("\n[DEBUG]\nUsing $mib\n[/DEBUG]\n");
+            echo ("\n[DEBUG]\n List Intance only names\n$listIntance\n[/DEBUG]\n");
+        }
+
```

Comment:
You've got a lot of redundant empty lines here, can you clean them down to single lines please? This goes for the rest of the file.